pysph/examples/flow_past_cylinder_2d.py

```diff
@@ -15,43 +8,56 @@
 from pysph.sph.equation import Equation
 from pysph.base.utils import get_particle_array
 from pysph.solver.application import Application
-from pysph.sph.scheme import SchemeChooser
-from pysph.sph.wc.edac import EDACScheme
 from pysph.tools import geometry as G
-from pysph.sph.wc.edac import SolidWallPressureBC
-from pysph.sph.bc.simple_inlet_outlet import SimpleInletOutlet
+from pysph.sph.wc.edac import EDACScheme
 from pysph.sph.bc.inlet_outlet_manager import (
         InletInfo, OutletInfo)
 
-# Geometric parameters
-l_tunnel = 15.0
-w_tunnel = 5.0
-diameter = 2.0  # diameter of circle
-center = l_tunnel / 3., 0.0  # center of circle
-n_inlet = 5  # Number of inlet layers
-n_outlet = 5  # Number of outlet layers
-n_wall = 5
 # Fluid mechanical/numerical parameters
-
 rho = 1000
 umax = 1.0
 c0 = 10 * umax
 p0 = rho * c0 * c0
 
 
-class ExtrapolateUhat(Equation):
-    def initialize(self, d_idx, d_uhat, d_wij):
-        d_uhat[d_idx] = 0.0
-        d_wij[d_idx] = 0.0
+class SolidWallNoSlipBCReverse(Equation):
```

Comment:
Is this the correct way to find the forces?  Does this need to be updated from the other repos?

Reply:
Yes, It is the same method used in inlet outlet repository.